js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,30 @@
+const handler = {
+  get(target, prop) {
+    if (prop in target) {
+      return target[prop];
+    } else if (typeof prop !== "symbol") {
+      const index = Number(prop);
+      if (index < 0) {
+        return target[index + target.length];
+      }
+      return target[index];
+    }
+  },
+  set(target, prop, newVal) {
+    const index = Number(prop);
+    if (index < 0) {
+      target[index + target.length] = newVal;
+      return true;
+    }
+    target[index] = newVal;
+    return true;
```

Comment:
I think a check like `if (typeof property !== "string") ` should be here before trying to convert the `prop` to `number`

Reply:
Maybe something like this:
` if (typeof property !== "string") {`
`       return Reflect.set(target, property, value, receiver);`
 ` }`
  ` const index = Number(property);`
   `   if (Number.isNaN(index)) {`
   `     return Reflect.set(target, property, value, receiver);`
   `  }`
   `  target[index < 0 ? target.length + index : index] = value;`

   `  return true;`